concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -406,7 +406,6 @@ jobs:
                 paas-cf/concourse/scripts/s3init.sh "((state_bucket))" google-oauth-secrets.yml paas-cf/concourse/init_files/zero_bytes
                 paas-cf/concourse/scripts/s3init.sh "((state_bucket))" cf.tfstate paas-cf/concourse/init_files/terraform.tfstate.tpl
                 paas-cf/concourse/scripts/s3init.sh "((state_bucket))" cf-certs.tar.gz paas-cf/concourse/init_files/empty.tar.gz
-                paas-cf/concourse/scripts/s3init.sh "((state_bucket))" cf-certs.tfstate paas-cf/concourse/init_files/terraform.tfstate.tpl
```

Comment:
Removing this line is going to cause the pipeline to stall in the `delete-old-cf-certs` task for new dev deployments. I think we should leave this in place until the cleanup PR.

Reply:
:thinking: It is true for new deployments. I would do a different course of action, and completely remove the usage of a resource. I would implement this as a task that downloads the file using S3.